onnx/defs/schema.cc

```diff
@@ -665,16 +674,23 @@ void OpSchema::ParseAndSetTypes(
   }
 }
 
-OpSchema& OpSchema::SetContextDependentFunctionBodyBuilder(ContextDependentFunctionBodyBuilder functionBuilder) {
-  functionBuilder_ = std::move(functionBuilder);
+OpSchema& OpSchema::SetContextDependentFunctionBodyBuilder(ContextDependentFunctionBodyBuilder functionBuilder, int opset_version) {
+  opset_version_to_function_builder_[opset_version] = std::move(functionBuilder);
```

Comment:
What happens if opset_version is -1 ?

Reply:
It will still be inserted. Eventually the functionBuilder for opset_version of -1 will be moved to since_version_ when since_version_ is updated in SinceVersion()
I am using -1 (static constexpr int kUninitializedSinceVersion = -1;) for uninitialized since_version_ and default opset_version value.